package.json

```diff
@@ -47,6 +47,8 @@
   "devDependencies": {
     "@babel/core": "7.14.3",
     "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@percy/cli": "^1.0.0-beta.50",
```

Comment:
beta 55 is the latest version and fixes a bunch of bugs, highly recommend updating